eZ/Publish/Core/MVC/Symfony/View/Builder/ViewBuilderRegistry.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\MVC\Symfony\View\Builder;
+
+/**
+ * A simple registry of ViewBuilders that uses the ViewBuilder's match() method to identify the builder.
+ */
+class ViewBuilderRegistry
```

Comment:
I should. I just have big issues naming the implementation and the interface in those cases ;-)